src/bundle/Resources/views/themes/standard/storybook/components/badge.html.twig

```diff
@@ -0,0 +1,5 @@
+{% extends '@IbexaDesignSystemStorybook/themes/standard/storybook/base_preview.html.twig' %}
+
+{% block component %}
```

Comment:
If you don't make any adjustments to properties, just passing all args to component, you can omit this file (and PR).